recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch

```diff
@@ -0,0 +1,39 @@
+From 488a3e0eee948cec2a6f87652bc3cea2a2d52e4f Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Thu, 1 Dec 2016 21:27:21 -0800
+Subject: [PATCH] resize-helper.service: Use ConditionFirstBoot directive
+
+ConditionFirstBoot when set to yes would be run
+when booting for first time. There is no need to
+manually enabled/disable the service then
+
+https://www.freedesktop.org/software/systemd/man/systemd.unit.html#ConditionFirstBoot=
+
+and reboot immediately after resize is done
+
+Signed-off-by: Khem Raj <raj.khem@gmail.com>
+---
+ resize-helper.service | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/resize-helper.service b/resize-helper.service
+index c1354a6..1a21cf7 100644
+--- a/resize-helper.service
++++ b/resize-helper.service
+@@ -1,11 +1,12 @@
+ [Unit]
+ Description=Resize root filesystem to fit available disk space
+ After=systemd-remount-fs.service
++ConditionFirstBoot=yes
+ 
+ [Service]
+ Type=oneshot
+ ExecStart=-/usr/sbin/resize-helper
+-ExecStartPost=/bin/systemctl disable resize-helper.service
++ExecStartPost=/bin/systemctl reboot
```

Comment:
I dont have reference to bugs to cite here but I think current approach is equally problematic ( i.e. live resize ) that reboot or not would not matter as much we need to reload the partition table after fdisking and I see that we are calling partprobe for this and then resize2fs is called so apps get the revised size when probed.

I will send a v2 without causing a reboot.